jujugui/static/gui/src/app/components/inspector-config/inspector-config.js

```diff
@@ -302,8 +311,12 @@ YUI.add('inspector-config', function() {
           <div className="inspector-config__fields">
             {this._customizeServiceName()}
             <form ref="file-form">
-              <input type="file" ref="file" className="hidden"
-                onChange={this._importConfig} />
+              <input
+                className="hidden"
+                disabled={disabled}
```

Comment:
I think you can have a disabled attribute for files, either way the button that opens this has been disabled too.